eslint.config.js

```diff
@@ -19,6 +19,7 @@ export default tseslint.config(
 			"test/tmp/*",
 			"test/projects/*",
 			"test/fixtures/*",
+			"test/e2e/*",
```

Comment:
Why shouldn't we lint the e2e tests? Only test fixtures and tmp files should be excluded

Reply:
Because of the imports of itself `test/e2e/package-exports.ts` must be excluded from `eslint` and `typescript` as they got confused by it. I'll exclude just this file, then.

Reply:
I have added some comments in the code why we need to exclude this file in eslint